### PR TITLE
18 integrations - Default args fix

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/33076
related: https://github.com/demisto/etc/issues/32855
related: https://github.com/demisto/etc/issues/32208

## Description
see in referenced issue

## Does it break backward compatibility?
   - [x] No

### CODE REVIEW

File: Packs/WootCloud/Integrations/WootCloud/WootCloud.yml [hunk 1]
```diff
@@ -62,31 +62,35 @@ script:
       - name: date_range
         description: Examples are (2 hours, 4 minutes, 6 month, 1 day, etc.)
         isArray: false
-        default: true
+        default: false
+        defaultValue: '1 day'
```
Comment (yaakovi) [inline Packs/WootCloud/Integrations/WootCloud/WootCloud.yml:66]: why did you add the default value?

File: Packs/WootCloud/Integrations/WootCloud/WootCloud.yml [hunk 2]
```diff
@@ -319,31 +323,34 @@ script:
...
```
Comment (yaakovi) [inline Packs/WootCloud/Integrations/WootCloud/WootCloud.yml:327]: same

File: Packs/WootCloud/Integrations/WootCloud/WootCloud.yml [hunk 3]
```diff
@@ -423,31 +430,34 @@ script:
...
```
Comment (yaakovi) [inline Packs/WootCloud/Integrations/WootCloud/WootCloud.yml:434]: same

File: Packs/WootCloud/Integrations/WootCloud/WootCloud.yml (hunk 1 above)
Comment (bakatzir) [reply Packs/WootCloud/Integrations/WootCloud/WootCloud.yml:66]: As I think that having a default value for arguments in general is a good idea.
But on a 2nd look I see that this arg is req.

Should I rm or keep?

**Comment by bakatzir** [comment]
pcap miner fails on timeout, non related IMO.
```
[2021-01-26 07:12:49] - [Thread-1] - [INFO] - Investigation URL: https://ec2-44-242-175-89.us-west-2.compute.amazonaws.com/#/WorkPlan/18
[2021-01-26 07:13:06] - [Thread-1] - [ERROR] - "PCAP Analysis Test" failed with error/s
[2021-01-26 07:13:06] - [Thread-1] - [ERROR] - Playbook "PCAP Analysis Test" has failed:
[2021-01-26 07:13:06] - [Thread-1] - [ERROR] - - Task ID: 170
[2021-01-26 07:13:06] - [Thread-1] - [ERROR] -   Command: !PcapMinerV2 entry_id="4@18" protocol_output="LANMAN,SRVSVC,SMB2,BROWSER,DCERPC,SMB,_WS.MALFORMED" extract_strings="True" pcap_filter="(smb or smb2)" custom_regex="" convs_to_display="15" wpa_password="" extract_ips="False"
[2021-01-26 07:13:06] - [Thread-1] - [ERROR] -   Body:
Error from Scripts is : Script failed to run: Runner request "run once" timeout reached for script [PcapMinerV2]. timeout: [10s] (2618) (2603)
[2021-01-26 07:13:06] - [Thread-1] - [ERROR] - - Task ID: 55
[2021-01-26 07:13:06] - [Thread-1] - [ERROR] -   Command: !PcapMinerV2 entry_id="4@18" pcap_filter="(smb or smb2)" filtered_file_name="filtered.pcap" convs_to_display="15" wpa_password="" extract_ips="True"
[2021-01-26 07:13:06] - [Thread-1] - [ERROR] -   Body:
Error from Scripts is : Script failed to run: Runner request "run once" timeout reached for script [PcapMinerV2]. timeout: [10s] (2618) (2603)
[2021-01-26 07:13:06] - [Thread-1] - [ERROR] - Test failed: playbook: "PCAP Analysis Test" with integration(s): ["ipinfo", "WildFire-v2"]
```

File: Packs/WootCloud/Integrations/WootCloud/WootCloud.yml (hunk 1 above)
Comment (yaakovi) [reply Packs/WootCloud/Integrations/WootCloud/WootCloud.yml:66]: adding default value makes the `required` flag redundant as it will always have a value.
if you think that this value is a common case - leave it. otherwise, remove it.

**Review by yaakovi — APPROVED** [review]

File: Packs/WootCloud/Integrations/WootCloud/WootCloud.yml (hunk 1 above)
Comment (bakatzir) [reply Packs/WootCloud/Integrations/WootCloud/WootCloud.yml:66]: I see. will remove as i'm not familiar with the product enough to set a defaultvalue.

File: Packs/ANYRUN/ReleaseNotes/1_0_1.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### ANY.RUN
+- Fixed an issue where a few arguments were marked as default arguments although they should not.
```
Comment (amshamah419) [inline Packs/ANYRUN/ReleaseNotes/1_0_1.md:4]:
```suggestion
- Fixed an issue where some arguments were incorrectly marked as default arguments.
```

File: Packs/AnsibleTower/ReleaseNotes/1_0_1.md [hunk 5]
```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### Ansible Tower
+- Fixed an issue where a few arguments were marked as default arguments although they should not.
```
Comment (amshamah419) [inline Packs/AnsibleTower/ReleaseNotes/1_0_1.md:3]:
```suggestion
- Fixed an issue where some arguments were incorrectly marked as default arguments.
```

File: Packs/ComputerVisionEngine/ReleaseNotes/1_0_1.md [hunk 6]
```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### Computer Vision Engine
+- Fixed an issue where a few arguments were marked as default arguments although they should not.
```
Comment (amshamah419) [inline Packs/ComputerVisionEngine/ReleaseNotes/1_0_1.md:3]:
```suggestion
- Fixed an issue where some arguments were incorrectly marked as default arguments.
```

File: Packs/Darktrace/ReleaseNotes/1_0_2.md [hunk 7]
```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### Darktrace
+- Fixed an issue where a few arguments were marked as default arguments although they should not.
```
Comment (amshamah419) [inline Packs/Darktrace/ReleaseNotes/1_0_2.md:3]:
```suggestion
- Fixed an issue where some arguments were incorrectly marked as default arguments.
```

File: Packs/ExpanseV2/ReleaseNotes/1_0_6.md [hunk 8]
```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### Expanse Expander Feed
+- Fixed an issue where a few arguments were marked as default arguments although they should not.
```
Comment (amshamah419) [inline Packs/ExpanseV2/ReleaseNotes/1_0_6.md:3]:
```suggestion
- Fixed an issue where some arguments were incorrectly marked as default arguments.
```

File: Packs/Forescout/ReleaseNotes/1_0_2.md [hunk 9]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Forescout
+- Fixed an issue where a few arguments were marked as default arguments although they should not.
```
Comment (amshamah419) [inline Packs/Forescout/ReleaseNotes/1_0_2.md:4]:
```suggestion
- Fixed an issue where some arguments were incorrectly marked as default arguments.
```

File: Packs/Graylog/ReleaseNotes/1_0_1.md [hunk 10]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+##### Graylog
+- Fixed an issue where a few arguments were marked as default arguments although they should not.
```
Comment (amshamah419) [inline Packs/Graylog/ReleaseNotes/1_0_1.md:4]:
```suggestion
- Fixed an issue where some arguments were incorrectly marked as default arguments.
```

File: Packs/Maltiverse/ReleaseNotes/1_0_2.md [hunk 11]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Maltiverse
+- Fixed an issue where a few arguments were marked as default arguments although they should not.
```
Comment (amshamah419) [inline Packs/Maltiverse/ReleaseNotes/1_0_2.md:4]:
```suggestion
- Fixed an issue where some arguments were incorrectly marked as default arguments.
```

File: Packs/McAfee_ESM/ReleaseNotes/1_1_3.md [hunk 12]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### McAfee ESM v2
+- Fixed an issue where a few arguments were marked as default arguments although they should not.
```
Comment (amshamah419) [inline Packs/McAfee_ESM/ReleaseNotes/1_1_3.md:4]:
```suggestion
- Fixed an issue where some arguments were incorrectly marked as default arguments.
```

File: Packs/MicrosoftGraphDeviceManagement/ReleaseNotes/1_0_6.md [hunk 13]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Microsoft Graph Device Management (Microsoft Intune)
+- Fixed an issue where a few arguments were marked as default arguments although they should not.
```
Comment (amshamah419) [inline Packs/MicrosoftGraphDeviceManagement/ReleaseNotes/1_0_6.md:4]:
```suggestion
- Fixed an issue where some arguments were incorrectly marked as default arguments.
```

File: Packs/RiskSense/ReleaseNotes/1_0_4.md [hunk 14]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### RiskSense
+- Fixed an issue where a few arguments were marked as default arguments although they should not.
```
Comment (amshamah419) [inline Packs/RiskSense/ReleaseNotes/1_0_4.md:4]:
```suggestion
- Fixed an issue where some arguments were incorrectly marked as default arguments.
```

File: Packs/ServiceNow/ReleaseNotes/2_1_7.md [hunk 15]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### ServiceNow CMDB
+- Fixed an issue where a few arguments were marked as default arguments although they should not.
```
Comment (amshamah419) [inline Packs/ServiceNow/ReleaseNotes/2_1_7.md:4]:
```suggestion
- Fixed an issue where some arguments were incorrectly marked as default arguments.
```

File: Packs/Sixgill-Darkfeed/ReleaseNotes/1_2_4.md [hunk 16]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Sixgill DarkFeed Enrichment
+- Fixed an issue where a few arguments were marked as default arguments although they should not.
```
Comment (amshamah419) [inline Packs/Sixgill-Darkfeed/ReleaseNotes/1_2_4.md:4]:
```suggestion
- Fixed an issue where some arguments were incorrectly marked as default arguments.
```

File: Packs/Snowflake/ReleaseNotes/1_0_1.md [hunk 17]
```diff
@@ -0,0 +1,4 @@
+#### Integrations
+##### Snowflake
+- Fixed an issue where a few arguments were marked as default arguments although they should not.
```
Comment (amshamah419) [inline Packs/Snowflake/ReleaseNotes/1_0_1.md:3]:
```suggestion
- Fixed an issue where some arguments were incorrectly marked as default arguments.
```

File: Packs/WootCloud/ReleaseNotes/1_0_3.md [hunk 18]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### WootCloud
+- Fixed an issue where a few arguments were marked as default arguments although they should not.
```
Comment (amshamah419) [inline Packs/WootCloud/ReleaseNotes/1_0_3.md:4]:
```suggestion
- Fixed an issue where some arguments were incorrectly marked as default arguments.
```

File: Packs/Workday/ReleaseNotes/1_0_7.md [hunk 19]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Workday
+- Fixed an issue where a few arguments were marked as default arguments although they should not.
```
Comment (amshamah419) [inline Packs/Workday/ReleaseNotes/1_0_7.md:4]:
```suggestion
- Fixed an issue where some arguments were incorrectly marked as default arguments.
```

File: Packs/Zscaler/ReleaseNotes/1_1_1.md [hunk 20]
```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### Zscaler
+- Fixed an issue where a few arguments were marked as default arguments although they should not.
```
Comment (amshamah419) [inline Packs/Zscaler/ReleaseNotes/1_1_1.md:3]:
```suggestion
- Fixed an issue where some arguments were incorrectly marked as default arguments.
```

File: Packs/mnemonicMDR/ReleaseNotes/1_0_1.md [hunk 21]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+##### mnemonic MDR - Argus Managed Defence
+- Fixed an issue where a few arguments were marked as default arguments although they should not.
```
Comment (amshamah419) [inline Packs/mnemonicMDR/ReleaseNotes/1_0_1.md:4]:
```suggestion
- Fixed an issue where some arguments were incorrectly marked as default arguments.
```

**Comment by bakatzir** [comment]
Failed on Maltiverse due to quota limit. force merging.
